apps/settings/lib/Controller/AppSettingsController.php

```diff
@@ -245,9 +245,11 @@ private function fetchApps() {
 		$apps = $appClass->listAllApps();
 		foreach ($apps as $app) {
 			$app['installed'] = true;
+			$appScreenshot = $app['screenshot'][0];
```

Comment:
there are apps without screenshots, so the isset is needed …

Or we make it default to null now as there is a `is_string` check afterwards
```suggestion
			$appScreenshot = $app['screenshot'][0] ?? null;
```

Reply:
Thanks for catching this, I took your suggestion